backend/src/model/__tests__/loadAllChatsModels.spec.ts

```
@@ -0,0 +1,90 @@
+import * as fs from 'fs';
+import { ConfigLoader } from '../../config/config-loader';
+import { ModelDownloader } from '../model-downloader';
+import { downloadAllModels } from '../utils';
+import { getConfigDir, getConfigPath } from 'src/config/common-path';
+
+const originalIsArray = Array.isArray;
+
+Array.isArray = jest.fn((type: any): type is any[] => {
+  if (
+    type &&
+    type.constructor &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)